src/packages/README.md

```diff
@@ -9,17 +9,90 @@ They have different levels of completeness, maturity, security, and support.
 release, which differs substantially from prior releases and software
 snapshots due to the introduction of a `pkg64c` package set.**
 
+## Overview
+
+The following table presents an overview of available package managers in
+CheriBSD that are described in more details in consecutive sections.
+
+| ABI        | #      | Manager | Install path   | Suitable for     | Examples |
+|------------|--------|---------|----------------|------------------|----------|
+| Hybrid ABI | 20,000 | `pkg64` | `/usr/local64` | Day-to-day use   | bash<br/>gdb-cheri<br/>llvm-base<br/>vim |
+| CheriABI   | 8,000  | `pkg64c`| `/usr/local`   | Experimental use | dash<br/>git<br/>nano<br/>rsync |
+
+## Package managers in CheriBSD
+
+*Note:* As of this writing we only provide packages for Morello systems.
+We aim to add CHERI-RISC-V package sets in the near future.
+
+CheriBSD includes two package managers: `pkg64` for hybrid ABI packages and
+`pkg64c` for CheriABI packages. The FreeBSD package manager `pkg` is not
+available on CheriBSD. The `pkg` program was replaced with a script that prints
+information on available package managers and always returns the exit code 1
+making `pkg` commands invalid on CheriBSD:
+```
+# pkg
+Please use the ABI-specific pkg variant:
+* pkg64c for experimental software built for CheriABI;
+* pkg64 for stable software built for the hybrid ABI.
+
+Replace pkg with pkg64c or pkg64 in your command and try again.
+# echo $?
+1
+```
+
+We expect that `pkg64c` will be renamed to `pkg` in a future CheriBSD release.
```

Comment:
This seems like a lot of implementation detail for a tutorial. Changing this text doesn't seem like a high priority though.

Reply:
I removed the details describing the `pkg` script.